src/acquisition/dotnetAcquisition.ts

```diff
@@ -60,50 +58,69 @@ function initializeDotnetAcquire(): void {
-    initializeDotnetAcquire();
+function getStreamLogs(): string {
+    return eventStreamObservers.map(observer => observer.log).filter(log => !!log).join('\n'); // Use OS-independent newline
+}
+
+export async function dotnetAcquire(
+    version: string,
+    telemetryProperties: { [key: string]: string | undefined },
+    issueProperties: { [key: string]: string | undefined }
+): Promise<string> {
+    try {
+        initializeDotnetAcquire();
 
-    if (!version || version === 'latest') {
-        throw new Error(`Cannot acquire .NET Core version "${version}". Please provide a valid version.`);
+        if (!version || version === 'latest') {
+            throw new Error(`Cannot acquire .NET Core version "${version}". Please provide a valid version.`);
+        }
+        return await acquisitionWorker.acquire(version, telemetryProperties);
+    } finally {
+        issueProperties.dotnetAcquireLog = getStreamLogs();
     }
-    return acquisitionWorker.acquire(version, telemetryProperties);
 }
 
-export async function ensureDotnetDependencies(dotnetPath: string, args: string[], telemetryProperties: { [key: string]: string | undefined }): Promise<void> {
-    initializeDotnetAcquire();
+export async function ensureDotnetDependencies(
+    dotnetPath: string, args: string[],
+    telemetryProperties: { [key: string]: string | undefined },
+    issueProperties: { [key: string]: string | undefined }
+): Promise<void> {
+    try {
+        initializeDotnetAcquire();
 
-    if (os.platform() !== 'linux') {
-        // We can't handle installing dependencies for anything other than Linux
-        telemetryProperties.skipped = "true";
-        return;
-    }
+        if (os.platform() !== 'linux') {
+            // We can't handle installing dependencies for anything other than Linux
+            telemetryProperties.skipped = "true";
+        }
 
-    const result = cp.spawnSync(dotnetPath, args);
-    const installer = new DotnetCoreDependencyInstaller(ext.outputChannel);
-    if (installer.signalIndicatesMissingLinuxDependencies(result.signal)) {
-        telemetryProperties.signalIndicatesMissing = "true";
-        await installer.promptLinuxDependencyInstall(telemetryProperties, 'Failed to successfully run the language server.');
-    } else {
-        telemetryProperties.signalIndicatesMissing = "false";
+        const result = cp.spawnSync(dotnetPath, args);
+        const installer = new DotnetCoreDependencyInstaller(ext.outputChannel);
+        if (installer.signalIndicatesMissingLinuxDependencies(result.signal)) {
+            telemetryProperties.signalIndicatesMissing = "true";
+            await installer.promptLinuxDependencyInstall(telemetryProperties, 'Failed to successfully run the language server.');
+        } else {
+            telemetryProperties.signalIndicatesMissing = "false";
+        }
+    } finally {
+        issueProperties.dotnetAcquireLog = getStreamLogs();
```

Comment:
Place the acquire log into a telemetry issueProperties member where it will be added to any issues that the users submits (if an error occurred).